android_jni/README.md

```diff
@@ -31,14 +31,26 @@ $ export ANDROID_SDK_ROOT="/path/to/android/sdk"
 $ export ANDROID_NDK_HOME="/path/to/android/ndk"
 ```
 
-Step 3 - Checkout and build libgav1
+Step 3 - Checkout and build libgav1 or dav1d
 
 ```
 $ cd ext
 $ ./libgav1_android.sh "${ANDROID_NDK_HOME}"
 $ cd ..
 ```
 
+If you want to use dav1d instead:
+
+```
+$ cd ext
+$ ./dav1d_android.sh "${ANDROID_NDK_HOME}"
+$ cd ..
+```
+
+Update [CMakeLists.txt](avifandroidjni/src/main/jni/CMakeLists.txt) as follows:
```

Comment:
Do you need to update the file or just the command line?

Reply:
This is the jni specific cmake file and it has to be updated. It sets the variables for libgav1 by default. For dav1d, they have to be updated.